js/src/sdk/models/actions.ts

```diff
@@ -276,7 +276,7 @@ export class Actions {
     try {
       const { data: res } = await apiClient.actionsV2.advancedUseCaseSearch({
         query: {
-          apps: data.apps.join(","),
+          apps: data.apps?.join(","),
```

Comment:
Good fix for handling undefined `data.apps`. Consider adding a default value to handle empty string case: `data.apps?.join(",") || ""`. This would prevent potential issues with the API expecting a string value.